assignments/lambda-classes.js

```diff
@@ -1 +1,144 @@
 // CODE here for your Lambda Classes
+class Person {
+    constructor(props){
+        this.name = props.name;
+        this.age = props.age;
+        this.location = props.location;
+        this.gender = props.gender;
+    }
+    speak() {
+        console.log (`Hello my name is ${this.name}, I am from ${this.location}.`);
+    }
+}
+
+class Instructor extends Person {
+    constructor(instructorProps){
+        super(instructorProps);
```

Comment:
The parameter inside the constructor function doesn't have to have a unique name for each constructor. What you'll see most often (in every constructor) is just `props`.